test/server/servers/SockJSServer.test.js

```diff
@@ -84,9 +84,18 @@ describe('SockJSServer', () => {
         done();
       }, 3000);
     });
+
+    it('should not throw an exception when connection is null', () => {
+      expect(() => {
+        const cb = jest.fn();
```

Comment:
Sorry one more nitpick but could you move this `cb` definition above the `expect` line, then move `expect(cb.mock.calls[0]).toEqual([null, null]);` below the `not.toThrow`? Just to isolate the behavior we want to "not throw".

Reply:
nvm and I love it